sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosClientTelemetryConfig.java

```diff
@@ -122,8 +161,8 @@ String getClientCorrelationId() {
      * @param tagNames - a comma-separated list of tag names that should be considered
      * @return current CosmosClientTelemetryConfig
      */
-    public CosmosClientBuilder metricTagNames(String tagNames) {
-        if (Strings.isNullOrWhiteSpace(tagNames)) {
+    public CosmosClientTelemetryConfig metricTagNames(String[] tagNames) {
```

Comment:
```suggestion
    public CosmosClientTelemetryConfig metricTagNames(String... tagNames) {
```

so users can write `metricTagNames("foo", "bar") `